end_to_end/fraud_detection/1-data-prep-e2e.ipynb

```diff
@@ -4,27 +4,30 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # flow_output_path = "YOUR_PATH_HERE"
</pre><p>Style nit: use brackets to indicate the user should fill in a value, ie <code>"YOUR_PATH_HERE"</code> -&gt; <code>"&lt;YOUR_PATH_HERE&gt;"</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Fixed</p>